server/app/views/components/ToastFragment.html

```diff
@@ -43,3 +51,34 @@
     ></div>
   </th:block>
 </div>
+
+<!--/* Render the toasts used in the Review/Summary page */-->
+<div th:fragment="summaryToasts">
+  <!--/* Success toast */-->
```

Comment:
question (blocking):  Don't we use these same toasts on the homepage and in the application blocks?  If so, then maybe it would be better to call this fragment "eligibilityToasts"?  I like the idea of making such small components reusable, rather than limiting them to use on a specific page.

Reply:
yes and no. there's overlap, but for instance, the block edit page has the "eligible" toast but not the "not eligible" one, and the program index page shows the "not eligible" toast but not the "eligible" toast. Thus far I've made them reusable by having the basic success/warning/alert/error fragments. I'm thinking about simplifying them some since some of the arguments don't seem to vary among all calls, but I was thinking of waiting until I implemented all toasts across the app before I do that.

Reply:
The way you describe it basically just moves summaryToasts and blockEditToasts and other future ones out into the template files. I'd prefer to keep them out of the template just to keep the template files from blowing up, but I could be convinced.

My basic plan is to do the refactoring I think you have in mind once I have added toasts for all pages. Basically, I want to verify something like "ah all alert toasts we have use randomly generated ids and the same settings, so I can simplify those templates so it just looks like <div th:replace="~{this :: alert(${bannerMessage})}"></div> " and those templates get a lot simpler, but I want to triple check that there isn't one weird edge case where one alert is slightly different first. Does that make sense?

Reply:
Makes sense.  The refactoring can wait.